Lib/test/test_compileall.py

```diff
@@ -625,7 +625,8 @@ def f(self, ext=ext, switch=switch):
                 ['-m', 'compileall', '-q', self.pkgdir]))
             # Verify the __pycache__ directory contents.
             self.assertTrue(os.path.exists(self.pkgdir_cachedir))
-            expected = sorted(base.format(sys.implementation.cache_tag, ext)
+            expected = ['.gitignore'] + \
+                       sorted(base.format(sys.implementation.cache_tag, ext)
                               for base in ('__init__.{}.{}', 'bar.{}.{}'))
```

Comment:
Suggestion to avoid the continuation backslash:
```suggestion
            expected = ['.gitignore'] + sorted(
                base.format(sys.implementation.cache_tag, ext)
                for base in ('__init__.{}.{}', 'bar.{}.{}')
            )
```